theano/gof/wrapper.py

```diff
@@ -0,0 +1,419 @@
+"""
```

Comment:
I think this doc should be moved into the web doc that explain how to make an op with c code.